packages/dx-react-scheduler-material-ui/src/templates/appointment/appointment.jsx

```diff
@@ -6,10 +6,11 @@ import { withStyles } from '@material-ui/core/styles';
 const styles = ({ palette, typography, spacing }) => ({
   appointment: {
     overflow: 'hidden',
-    backgroundColor: palette.primary[300],
     boxSizing: 'border-box',
-    borderRight: `1px solid ${palette.background.paper}`,
-    borderBottom: `1px solid ${palette.background.paper}`,
+    borderRight: '1px solid transparent',
+    borderBottom: '1px solid transparent',
+    backgroundClip: 'padding-box',
+    backgroundColor: palette.primary[300],
```

Comment:
We had some problems with such colors, didn't we?

Reply:
Yes